exopy_hqc_legacy/tasks/tasks/instr/dc_tasks.py

```diff
@@ -12,11 +12,35 @@
 import time
 import numbers
 
-from atom.api import (Float, Value, Str, Int, set_default, Tuple)
+from atom.api import (Float, Value, Str, Int, set_default, Enum, Tuple)
 
 from exopy.tasks.api import (InstrumentTask, TaskInterface,
                             InterfaceableTaskMixin, validators)
 
+class GetDCVoltageTask(InstrumentTask):
+    """Get the current DC voltage of an instrument
+    """
+
+    parallel = set_default({'activated': False, 'pool': 'instr'})
+    database_entries = set_default({'voltage': 0.01})
+
+    def perform(self, value=None):
+        """Default interface.
+
+        """
+        if self.driver.owner != self.name:
+            self.driver.owner = self.name
+            if hasattr(self.driver, 'function') and\
+                    self.driver.function != 'VOLT':
+                msg = ('Instrument assigned to task {} is not configured to '
+                       'output a voltage')
+                raise ValueError(msg.format(self.name))
+
+        current_value = getattr(self.driver, 'voltage')
+        print('The current voltage is: {}'.format(current_value))
+        
+        self.write_in_database('voltage', float(current_value))
+
```

Comment:
When I try to use this task it prints the following warning:
WARNING: The profile stored in C:\Users\Marius\exopy\instruments\profiles\PXA.instr.ini does not declare the model field or it contains incorrect values : 'PXA'
And I cannot select any tool after clicking "Select" (impossible to set the tool name and model).

Adding the PXA is one of the things we did from my computer, I do not know where the conflict could have started

Reply:
Yes, I think it indeed comes from your installation because I can't reproduce this. Can you try with a fresh profile folder?

Reply:
Alright I think I understand better now, it is related to Exopy so has nothing to do with this PR on exopy_hqc_legacy.
I guess we did something wrong with the PXA driver we messes up with this new task. I guess we can figure this out after the PR

Reply:
Same bug with a fresh profile folder

Reply:
Actually this WARNING prints out anytime I want to select an instrument for any first task I am adding to a measurement, and it is non-critical for other tasks such as Set_DC_current. Only Get_DC_Voltage's tool selection is frozen out

Reply:
After discussing with @MVillius, we found that the issue was not related to this PR